### PR TITLE
Add sbt builder 1.0

### PR DESCRIPTION
**Pull request by BeLi4L**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,15 @@
+FROM codingame/sbt:0.13.13
+
+# Add "assembly" plugin, to be able to create a fat jar
+RUN mkdir -p ~/.sbt/0.13/plugins/ \
+	&& echo 'addSbtPlugin("com.eed3si9n" % "sbt-assembly" % "0.14.4")' \
```
Comment (fraisse) [inline Dockerfile]: What is this `0.14.4`?

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+		> ~/.sbt/0.13/plugins/assembly.sbt \
```
Comment (fraisse) [inline Dockerfile]: `~/.sbt/0.13/plugins` is duplicated. You maybe should create a variable?

File: Dockerfile (hunk 1 above)
Comment (fraisse) [inline Dockerfile]: Probably the sbt-assembly plugin version…

Comment (BeLi4L) [inline Dockerfile]: It's the version of sbt-assembly

File: Dockerfile (hunk 2 above)
Comment (BeLi4L) [inline Dockerfile]: Done in 2af9f25485520588f9cb0ee86bea60eb67ba52b8

**Pull request merged**